pom.xml

```diff
@@ -300,7 +300,10 @@
                                             <artifact>
                                                 <id>org.eclipse.winery:org.eclipse.winery.repository:2.0.0-SNAPSHOT</id>
                                                 <source>true</source>
-                                                <transitive>false</transitive>
+                                                <excludes>
+                                                    <exclude>org.slf4j:::</exclude>
+                                                    <exclude>org.eclipse.collections:eclipse-collections*::</exclude>
```

Comment:
Is it easy to say why these to libraries are causing issues? I would assume that the repository library depends on these two. Maybe, it is just a version inconsistency somewhere/somehow?

Reply:
the repository library transitively depends on them as **optional** deps.

They are declared as dependencies through the p2-site plugin, but not resolved through maven, and thus AFAICT not included in the generated site. This leads to OSGI errors. If the dependencies are explicitly marked as excluded in the p2-plugin definition, they're not declared as dependency and the problem does not show up.

For now that seems to work. A possibly safer solution would be to explicitly add these as dependencies we want to have in the site. I have not tested that solution, though.